src/components/miscellaneous/JobPostListItem.tsx

```diff
@@ -10,51 +10,50 @@ import Paper, { PaperProps } from 'components/layout/Paper';
 import AspectRatioImg, { AspectRatioLoading } from 'components/miscellaneous/AspectRatioImg';
 
 export type JobPostListItemProps = {
-  jobpost: JobPost;
+  jobPost: JobPost;
   sx?: PaperProps['sx'];
 };
 
-const JobPostListItem = ({ jobpost, sx }: JobPostListItemProps) => (
+const JobPostListItem = ({ jobPost, sx }: JobPostListItemProps) => (
   <ListItemButton
     component={Link}
     focusRipple
     sx={{ p: 0, overflow: 'hidden', borderRadius: (theme) => `${theme.shape.borderRadius}px`, ...sx }}
-    to={`${URLS.jobposts}${jobpost.id}/${urlEncode(jobpost.title)}/`}>
+    to={`${URLS.jobposts}${jobPost.id}/${urlEncode(jobPost.title)}/`}>
     <Paper noOverflow noPadding sx={{ width: '100%' }}>
+      <div style={{ display: 'block' }}>
+        <AspectRatioImg
+          alt={jobPost.image_alt || jobPost.title}
+          src={jobPost.image}
+          sx={{ height: 'auto', width: '100%', objectFit: 'cover', borderRadius: (theme) => `${Number(theme.shape.borderRadius) / 2}px` }}
+        />
+      </div>
```

Comment:
div'en rundt kan fjernes nå som aspect-ratio er støttet overalt
```suggestion
      <AspectRatioImg
        alt={jobPost.image_alt || jobPost.title}
        src={jobPost.image}
        sx={{ height: 'auto', width: '100%', objectFit: 'cover', borderRadius: (theme) => `${Number(theme.shape.borderRadius) / 2}px` }}
      />
```